index.bs

```diff
@@ -1404,13 +1404,7 @@ The {{MLGraphBuilder}} interface defines a set of operations as identified by th
 <script type=idl>
 typedef record<DOMString, MLOperand> MLNamedOperands;
 
-dictionary MLBufferResourceView {
-  required GPUBuffer resource;
```

Comment:
Because this PR and #546 remove all WebNN interfaces using `GPUBuffer` and `GPUTexture`, I suppose we should also modify [6.2. Device Selection](https://www.w3.org/TR/webnn/#programming-model-device-selection) section that overviews the usage of `GPUBuffer` and `GPUTexture` for `MLContext` created from a `GPUDevice`.

> In addition to the default method of creation with [MLContextOptions](https://www.w3.org/TR/webnn/#dictdef-mlcontextoptions), an [MLContext](https://www.w3.org/TR/webnn/#mlcontext) could also be created from a specific [GPUDevice](https://www.w3.org/TR/webgpu/#gpudevice) that is already in use by the application, in which case the corresponding [GPUBuffer](https://www.w3.org/TR/webgpu/#gpubuffer) resources used as graph constants, as well as the [GPUTexture](https://www.w3.org/TR/webgpu/#gputexture) as graph inputs must also be created from the same device.

> The following table summarizes the types of resource supported by the context created through different method of creation:

> Creation method | ArrayBufferView | GPUBuffer | GPUTexture
> -- | -- | -- | --
> MLContextOptions | Yes | No | No
> GPUDevice | Yes | Yes | Yes

Reply:
SGTM. Removed this table in the latest patch

Reply:
Thanks!